internal/core/textlayout/linebreaker.rs

```diff
@@ -119,6 +122,14 @@ impl<'a, Font: TextShaper> Iterator for TextLineBreaker<'a, Font> {
                 }
             };
 
+            if !fragment.trailing_mandatory_break
```

Comment:
I'm wondering, wouldn't it be easier to set `self.fragments.break_anywhere` to true in line 108/111? Before the `let mut next_line = loop {` loop, basically?

Reply:
Got it, thanks:). Could you add this as a comment maybe?:)